Source/CarthageKit/GitHub.swift

```diff
@@ -266,6 +284,14 @@ internal struct GitHubCredentials {
 	}
 }
 
+internal func loadGitHubCredentials(accessToken: GitHubAccessToken?) -> SignalProducer<GitHubCredentials?, CarthageError> {
```

Comment:
Oh, Swift. Maybe that will work in 2.0.

Let's go with my original suggestion of returning the authorization header value as a `String`.

Thanks for trying the `enum`!